hugo/assets/scss/components/separator.scss

```diff
@@ -19,4 +19,12 @@
     &--wide {
         max-width: $w-section-content + 120px;
     }
+
+    &--blog {
```

Comment:
For this ticket I will create both the article__footer and put the seperator HTML-element in it (since we don't want an empty article__footer with seperator-styling applied to it). 
Then indeed when we pick up the tags I will go for your solution.

Reply:
Sounds good!